src/coreComponents/dataRepository/Group.cpp

```diff
@@ -589,7 +589,7 @@ void Group::enableLogLevelInput()
     setDescription( "Log level" );
 }
 
-Group const & Group::getBaseGroupByPath( string const & path ) const
+Group const & Group::getBaseGroupByPath( string const & path, bool hardQuery ) const
```

Comment:
wouldn't `getGroupPointer()` be an option here?

Reply:
The behavior could probably just be replicated with `getGroupByPath` for the containing group which is assumed to exist and then check that the subgroup exists in the parent group.

Even with that workaround (assuming a group exists -- though there are many groups it is safe to assume do exist), we don't have a concise way of saying "does this arbitrary path exist in the data repository" without erroring out if it doesn't.